htdocs/projet/element.php

```diff
@@ -513,7 +512,7 @@
 {
 
 	$tablename = GETPOST("tablename", "aZ09");
-    $projectField = GETPOST("projectfield", "aZ09");
+    $projectField = GETPOST('projectfield', 'aZ09') ? GETPOST('projectfield', 'aZ09') : 'fk_projet';
```

Comment:
@lvessiller-opendsi 
better is : 

`$projectField = GETPOSTISSET('projectfield') ? GETPOST('projectfield', 'aZ09') : 'fk_projet';`

Reply:
Thanks for your remark, I changed this check.